### PR TITLE
Add USB and SWD connectors to schematics

### LINKED ISSUE #9: Add SWD / Debug connector

### LINKED ISSUE #15: Connector for Power
In a lab environment it's more practical to use a bench-top power supply than a battery.

### PR DESCRIPTION
**Pull request by dbrgn**
Fixes #15, fixes #9.

Necessary part for USB: https://www.digikey.ch/product-detail/en/molex/1051330001/WM9734CT-ND/4037910?cur=CHF&lang=en

### CODE REVIEW

**Comment by dbrgn** [comment]
Do we need to avoid feeding the battery directly into the USB host? Do we need to avoid sourcing power from USB and the battery in parallel?

**Comment by dbrgn** [comment]
Ok, ready for review!

File: circuit/circuit.lp [hunk 1]
```diff
@@ -356,6 +393,15 @@
   (name "GND3") (value "GND")
   (signal ff161c97-29a5-43aa-a9ae-3ca7a66982ce (net 7bce9215-22e4-4be8-9bec-7495ce622913))
  )
+ (component 6c1d9680-4b28-4ba5-b045-0dca29323524
+  (lib_component f7b52b62-7814-4a74-88ac-155e101507d5)
+  (lib_variant 75a3b2bb-300f-4ef0-9f8e-dbe84c4c4d55)
+  (lib_device f7139d7c-a7df-4cb2-af92-06b38f7a12f4)
+  (name "J7") (value "{{PARTNUMBER}}")
```
Comment (rnestler) [inline circuit/circuit.lp]: Can we add a comment to the value about that we use it as a selection jumper? Since we can't freely add text labels.

**Comment by dbrgn** [comment]
Segger Docs: https://www.segger.com/products/debug-probes/j-link/technology/interface-description/

File: circuit/circuit.lp (hunk 1 above)
Comment (dbrgn) [reply circuit/circuit.lp]: Done!

**Review by rnestler — APPROVED** [review]

**Pull request merged**